compact/tracking/vertex_barrel.xml

```diff
@@ -1,157 +1,248 @@
+    <constant name="Section_length" value="RSU_length/2-backbone_width"/>
 
-    <constant name="SiVertexSensor_thickness"           value="40*um"/>
+    <constant name="VertexBarrelMod_length"   value="RSU_length/2" />
+    <constant name="VertexBarrelLayer_nz"     value="12*2" />
+    <comment>
+      # of RSU in phi: 12, 16, 40 RSUs = 24, 32, 80 sections
+    </comment>
+    <constant name="VertexBarrelL0_nphi" value="12*2" />
+    <constant name="VertexBarrelL1_nphi" value="16*2" />
+    <constant name="VertexBarrelL2_nphi" value="40*2" />
 
+    <constant name="VertexBarrelStaveL0_width" value="RSU_width/2" />
+    <constant name="VertexBarrelStaveL1_width" value="RSU_width/2" />
+    <constant name="VertexBarrelStaveL2_width" value="RSU_width/2" />
 
-    <constant name="VertexBarrelMod1_rmin"              value="VertexBarrelMod_rmin"/>
-    <constant name="VertexBarrelMod2_rmin"              value="VertexBarrelMod_rmin + 1.2*cm"/>
-    <constant name="VertexBarrelMod3_rmin"              value="12.0*cm"/>
+    <comment>
+      the radius here include a gap b/w upper and lower half.
+      L0 and L1 values from drawing (see vertex_barrel_support.xml)
+    </comment>
+    <constant name="VertexBarrelMod_thickness" value="0.2*mm" />
+    <constant name="VertexBarrelModL0_rmin" value="38*mm" />
+    <constant name="VertexBarrelModL1_rmin" value="50.3*mm+1*um" />
+    <constant name="VertexBarrelModL2_rmin" value="125.2*mm+1*um" />
+    <!-- <constant name="VertexBarrelModL2_rmin" value="VertexBarrelL2_nphi*VertexBarrelStaveL2_width/2/Pi" /> -->
 
-    <comment> ensure we are within the vertex envelope with some margin. </comment>
-    <constant name="VertexCheck"                        value="sqrt(VertexBarrel_rmax - VertexBarrelMod3_rmin - .1*cm)"/>
+    <constant name="VertexBarrelMod_vert_off" value="1.0*mm" /> <comment>roughly (38.15*2pi-234.8)/4 </comment>
+    <constant name="VertexBarrelModL0_dphi"   value="VertexBarrelMod_vert_off/VertexBarrelModL0_rmin" />
+    <constant name="VertexBarrelModL1_dphi"   value="VertexBarrelMod_vert_off/VertexBarrelModL1_rmin" />
+    <constant name="VertexBarrelModL2_dphi"   value="VertexBarrelMod_vert_off/VertexBarrelModL2_rmin" />
 
     <documentation>
-      - Currently there are 3 sensor layers. Each is composed of 2 half-cylinders modules with only 40um of silicon thickness.
-      - Both support shells are 300um thick, implemented as the integrated tracker support/service setup
-
+      for the segmentation. Assume sensors are placed at the inner most surface for each layer
     </documentation>
+    <constant name="VertexBarrelSegL0_r" value="VertexBarrelModL0_rmin+SiVertexSensor_thickness/2.0" />
+    <constant name="VertexBarrelSegL1_r" value="VertexBarrelModL1_rmin+SiVertexSensor_thickness/2.0" />
+    <constant name="VertexBarrelSegL2_r" value="VertexBarrelModL2_rmin+SiVertexSensor_thickness/2.0" />
 
-    <constant name="VertexBarrelEnvelope_length"    value="VertexTrackingRegion_length"/>
```

Comment:
This causes
https://github.com/eic/epic/actions/runs/18144969161/job/51644821946?pr=803#step:5:2226
```
jinja2.exceptions.UndefinedError: 'VertexBarrelEnvelope_length' is undefined
```
Perhaps we can somehow provide some reasonable value for envelope length? Or we can discontinue its output in comparison template https://github.com/eic/epic/blob/fc2eb7a5f35264d46c5c5c04e0404e3b816b8e82/templates/DetectorParameterTable.csv.jinja2#L44-L46